docs/pages/pmd/userdocs/cli_reference.md

```diff
@@ -196,30 +205,33 @@ to be "debug".
 
 The language is determined automatically by PMD from the file extensions. Some languages such as "Java"
 however support multiple versions. The default version will be used, which is usually the latest supported
-version. If you want to use an older version, so that e.g. rules, that suggest usage of language features,
-that are not available yet, won't be executed, you need to specify a specific version via the `-language`
-and `-version` parameter.
-
-These parameters are irrelevant for languages that don't support different versions.
+non-preview version. If you want to use an older version, so that e.g. rules, that suggest usage of language features,
+that are not available yet, won't be executed, you need to specify a specific version via the `--use-version`
+parameter.
```

Comment:
German commas ^^
```suggestion
non-preview version. If you want to use an older version, so that e.g. rules that suggest usage of language features
that are not available yet won't be executed, you need to specify a specific version via the `--use-version`
parameter.
```